server/hedley/modules/custom/hedley_municipality/hedley_municipality.module

```diff
@@ -393,3 +393,37 @@ function hedley_municipality_get_non_empty_language_for_field(array $field_attri
     }
   }
 }
+
+/**
+ * Implements hook_date_format_types().
+ */
+function hedley_municipality_date_format_types() {
+  return ['municipality_date' => t('Municipality Date')];
+}
+
+/**
+ * Implements hook_date_formats().
+ */
+function hedley_municipality_date_formats() {
+  $formats = [];
+
+  $formats[] = [
+    'type' => 'municipality_date',
+    'format' => 'F j, Y',
+    'locales' => ['en'],
+  ];
+
+  $formats[] = [
+    'type' => 'municipality_date',
+    'format' => 'j בF Y',
```

Comment:
For hebrew we add the `ב` just before the month label as in the issue:
![new_theme__news_item_page__4h__ _issue__336_ _gizra_municipality](https://user-images.githubusercontent.com/5985683/27261679-95472cdc-5450-11e7-9dbf-0fcb918172ba.gif)

and it actually works pretty good:
![municipality](https://user-images.githubusercontent.com/5985683/27261695-fbe2458a-5450-11e7-95fd-6058afac1f02.gif)